cli/dcoscli/package/main.py

```diff
@@ -488,3 +493,36 @@ def _uninstall(package_name, remove_all, app_id, cli, app):
         return 1
 
     return 0
+
+def __confirm_uninstall(package_name, remove_all, app_id):
```

Comment:
Double underscore is weird.  This is a pythonism for "mangle this symbol to do fake data hiding", which probably shouldn't have been added to the language in the first place.
Single underscore is a convention for "don't show this object to people importing this module the normal way".